build-info-extractor-maven3/src/main/java/org/jfrog/build/extractor/maven/BuildDeploymentHelper.java

```diff
@@ -48,26 +36,26 @@
     @Requirement
     private ArtifactoryManagerBuilder artifactoryManagerBuilder;
 
-    public void deploy( Build                          build,
-                        ArtifactoryClientConfiguration clientConf,
-                        Map<String, DeployDetails>     deployableArtifactBuilders,
-                        boolean                        wereThereTestFailures,
-                        File                           basedir ) {
+    public void deploy(BuildInfo buildInfo,
+                       ArtifactoryClientConfiguration clientConf,
+                       Map<String, DeployDetails> deployableArtifactBuilders,
+                       boolean wereThereTestFailures,
+                       File basedir) {
```

Comment:
Let's use this opportunity and align the above code lines according to our conventions.